### ISSUE TITLE
Fix `shorthand-property-no-redundant-values` false negatives for additional radius

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#8428]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  border-radius: 1px 1px 1px 1px / 2px 2px 2px 2px;
}
```

I encountered this while looking into https://github.com/stylelint/stylelint/issues/8412. I couldn't find it coming up before, I guess because additional radii are rarely used.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
     "shorthand-property-no-redundant-values": true
   }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLgIwgCdZCBaQpKASwFcBneFARgAcAPZ9zj1jgehQAmLsI6ih7ANxYAvlhAAaEADMqAGxgA5JAFs4iGG10sNAOjB06i8BEyqA5ghAZMKFOhCGcMTFDofGAG0sNzcPOhwATw01KkwcUkg7KntSCKRfJGIPEJQAXVlrJIcAMSIdJBwnACs6W2tYFitEFzCQCOiYWPiA9xA1SpgIjwVc8KiYuITilLScDKgsqF6PAe9hkEKZIA).

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

Both sets of redundant values in the radii to be reported.

### What actually happened?

No problems reported.

### Do you have a proposal to fix the bug?

Account for additional radius.

**Comment by jeddy3** [comment]
As there are no objections, I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
